Task4/Task4/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Task4
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значение x");
+            var ics = double.Parse(Console.ReadLine());
+            var y = calculation(ics);
+            Console.WriteLine("f(x) = " + y);
+            Console.ReadKey();
+        }
+
+        static double calculation(double x)
+        {
+            double y = 2 * Math.Sin(Math.Pow(Math.Pow(x,2) + 4, 0.5) / 2) * Math.Cos(Math.Pow(Math.Pow(x, 2) + 1, 0.5) / 2);
```

Comment:
Место ` Math.Power(smth, 0.5)` лучше использовать `Math.Sqrt(smth)`
```suggestion
            double y = 2 * Math.Sin(Math.Sqrt(Math.Pow(x,2) + 4) / 2) * Math.Cos(Math.Sqrt(Math.Pow(x, 2) + 1) / 2);
```